File: SoloLearn Codes/Security/Security.java
import java.util.Scanner;

public class Security
{
    public static void main(String[] args) {
        String data = new Scanner(System.in).nextLine();
        String ft = data.replace("x","");
        int thief = ft.indexOf('T');
        System.out.println((ft.charAt(thief + 1 >= ft.length() ? thief : thief + 1) == '$') || (ft.charAt(thief - 1 < 0 ? thief : thief - 1) == '$') ? "ALARM" : "quiet");
    }
}

File: SoloLearn Codes/Security/Security.cs
using System;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            string s = Console.ReadLine().Replace("x", "");
            Console.WriteLine(s.IndexOf("T$") >= 0 || s.IndexOf("$T") >= 0 ? "ALARM" : "quiet");
        }
    }
}